### DISCUSSION TITLE
Using Fonts in Freeplane

### DISCUSSION THREAD

**Discussion by olazzzzz** [freeplane/freeplane#957] [Help] [answered]
Dear Freeplane community:

Loyal Freeplane user here (although not very tech-savvy), writing its first post!
Thank you all for this wonderful program, it has contributed to my personal and professional life in so many valuable ways!

I cannot quite understand how to install new fonts to use in Freeplane. From an old post, I understood that they have to be installed in Windows/fonts but also on the Java directory. Using this method, some of the fonts I installed work (Roboto, DIN, Filson Pro), some don't (Satoshi, Iosevka, Zodiak). Is there a reason for this or a workaround?

Thank you all again

**Comment by macmarrum** [comment, marked as answer]
Hi @olazzzzz

It's great to hear from you.

Regarding fonts, please have a look at
- https://github.com/freeplane/freeplane/issues/802
